example/integration_test/app_test.dart

```diff
@@ -87,5 +87,42 @@ void main() {
       expect(tokenText.data?.isNotEmpty, true);
       expect(tokenText.data?.toLowerCase().contains('error'), false);
     });
+
+    testWidgets('fetchInit and fetchExecute runs', (tester) async {
+      app.main();
+      await tester.pumpAndSettle();
```

Comment:
Now that we have two inits, please confirm if pumpAndSettle actually resets reCAPTCHA (I bet it doesn't), which means that the ordering of tests matter and fetchClient isn't actually triggering initialization.

Reply:
I will investigate this further in a different branch. This one both manual tests and integration are passing in both iOS and Android.